exercises/practice/binary/.meta/config.json

```diff
@@ -1,6 +1,24 @@
 {
   "blurb": "Convert a binary number, represented as a string (e.g. '101010'), to its decimal equivalent using first principles",
-  "authors": [],
+  "authors": [
+    "rubysolo"
+  ],
+  "contributors": [
+    "andrewsardone",
+    "angelikatyborska",
+    "Cohen-Carlisle",
+    "dalexj",
+    "devonestes",
+    "jinyeow",
+    "kytrinyx",
+    "lpil",
+    "neenjaw",
+    "NobbZ",
```

Comment:
I can't find myself in the commit history, can someone point me to the location? Even though I can't remember contributing, the combination from language and exercise makes it likely I indeed did :D

Reply:
I found at least three commits of yours for that exercise:
https://github.com/exercism/elixir/commit/b0d56986e140db7ac55d59082cfba7235fac43d3
https://github.com/exercism/elixir/commit/70d4e7c6a1926ca339ff5092c2abef69c000519f
https://github.com/exercism/elixir/commit/0eb33b7da1f7b35dc27703327cc10546e77f48d6